backend/src/main/java/com/google/rolecall/models/User.java

```diff
@@ -359,14 +359,14 @@ public Builder setEmail(String email) {
     }
   
     public Builder setNotificationEmail(String notificationEmail) {
-      if(email != null) {
+      if(notificationEmail != null) {
         this.notificationEmail = notificationEmail.toLowerCase();
       }
       return this;
     }
   
     public Builder setPictureFile(String pictureFile) {
-      if(email != null) {
+      if(pictureFile != null) {
         this.pictureFile = pictureFile.toLowerCase();
```

Comment:
I believe this should be made lowercase. in URL case is sensitive.

Reply:
Done.

Reply:
sorry i had a typo. I was saying this should NOT be made lowercase.

basically you can change to 
  this.pictureFile = pictureFile.
and remove the toLowerCase() call 

Thanks
Richard